src/databrowser.js

```diff
@@ -165,7 +165,7 @@ define([
                 }
                 info = info._parent;
             }
-            vellum.handleDropFinish(target, path);
+            vellum.handleDropFinish(target, path, widget.mug);
```

Comment:
Fixed. You can probably ignore this PR for now. I'm going to crank out tests for this stuff so I feel better putting it on prod. Will ping once done